dockerfiles/che/entrypoint.sh

```diff
@@ -99,10 +98,8 @@ set_environment_variables () {
     CHE_HOME=$(echo /"${CHE_HOME}" | sed  's|\\|/|g' | sed 's|:||g')
   fi
 
-  # Che configuration directory - where che.properties lives
-  if [ -z "${CHE_LOCAL_CONF_DIR}" ]; then
-    export CHE_LOCAL_CONF_DIR="${CHE_HOME}/conf/"
-  fi
+  # All Eclipse Che variables has to be set as environment variables.
+  unset CHE_LOCAL_CONF_DIR
```

Comment:
question why do we need to unset if it's not used ?

Reply:
To make sure nobody trying to work around it and we have perfect portability.
BTW If it's blocker for you I can remove it.

Reply:
Im ok to remove that unset

Reply:
ok. No problem